.cirrus.yml

```diff
@@ -5,39 +5,40 @@ freebsd_instance:
 # same VM.  The binary will be built in 32-bit mode, but will execute on a
 # 64-bit kernel and in a 64-bit environment.  Our tests don't execute any of
 # the system's binaries, so the environment shouldn't matter.
-# task:
-#   name: FreeBSD 12.0
-#   env:
-#     LOOM_MAX_DURATION: 10
-#   setup_script:
-#     - pkg install -y curl
-#     - curl https://sh.rustup.rs -sSf --output rustup.sh
-#     - sh rustup.sh -y
-#     - . $HOME/.cargo/env
-#     - rustup target add i686-unknown-freebsd
-#     - |
-#       # Remove any existing patch statements
-#       mv Cargo.toml Cargo.toml.bck
-#       sed -n '/\[patch.crates-io\]/q;p' Cargo.toml.bck > Cargo.toml
-#
-#       # Patch all crates
-#       cat ci/patch.toml >> Cargo.toml
-#
-#       # Print `Cargo.toml` for debugging
-#       echo "~~~~ Cargo.toml ~~~~"
-#       cat Cargo.toml
-#       echo "~~~~~~~~~~~~~~~~~~~~"
-#   cargo_cache:
-#     folder: $HOME/.cargo/registry
-#   test_script:
-#     - . $HOME/.cargo/env
-#     - cargo test --all
-#     - (cd tokio-trace/test-log-support && cargo test)
-#     - (cd tokio-trace/test_static_max_level_features && cargo test)
-#     - cargo doc --all
-#   i686_test_script:
-#     - . $HOME/.cargo/env
-#     - |
-#       cargo test --all --exclude tokio-tls --exclude tokio-macros --target i686-unknown-freebsd
-#   before_cache_script:
-#     - rm -rf $HOME/.cargo/registry/index
+task:
+  name: FreeBSD 12.0
+  env:
+    LOOM_MAX_DURATION: 10
+  setup_script:
+    - pkg install -y curl
+    - curl https://sh.rustup.rs -sSf --output rustup.sh
+    - sh rustup.sh -y --default-toolchain nightly
+    - . $HOME/.cargo/env
+    - rustup target add i686-unknown-freebsd
+    - |
+      # Remove any existing patch statements
+      mv Cargo.toml Cargo.toml.bck
+      sed -n '/\[patch.crates-io\]/q;p' Cargo.toml.bck > Cargo.toml
+
+      # Patch all crates
+      cat ci/patch.toml >> Cargo.toml
+
+      # Print `Cargo.toml` for debugging
+      echo "~~~~ Cargo.toml ~~~~"
+      cat Cargo.toml
+      echo "~~~~~~~~~~~~~~~~~~~~"
+  cargo_cache:
+    folder: $HOME/.cargo/registry
+  test_script:
+    - . $HOME/.cargo/env
+    - cargo test --all --lib && cargo test --all --tests
+    - (cd tokio-trace/test-log-support && cargo test)
+    - (cd tokio-trace/test_static_max_level_features && cargo test)
+    - cargo doc --all
+  # TODO: Re-enable
```

Comment:
is there a reason for leaving this out right now?

Reply:
Mostly to avoid having to rewrite it temporarily :) We can always fix on master.